powerquery-docs/connectors/cloudbluepsa.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Power Query CloudBlue PSA connector
+description: Provides basic information and prerequisites for the operation of the CloudBlue PSA connector in conjuction with the API.
+author: mohsad06
+ms.topic: conceptual
+ms.date: 3/7/2023
+ms.author: bezhan 
+---
+
+# CloudBluePSA (Beta)
+
+>[!Note]
+The following connector article is provided by CloudBluePSA, the owner of this connector and a member of the Microsoft Power Query Connector Certification Program. If you have questions regarding the content of this article or have changes you would like to see made to this article, visit the CloudBluePSA website and use the support channels there.
+
+## Summary
+
+| Item | Description |
+| ---- | ----------- |
+| Release State | General Availability |
```

Comment:
This connector appears to be a "Beta", so change "General Availability" to "Beta".